build.rs

```diff
@@ -43,8 +43,8 @@ fn main() {
                .define("ENABLE_MODULE_RECOVERY", Some("1"));
 
     // secp256k1
-    base_config.file("depend/secp256k1/src/laxder_shim.c")
-               .file("depend/secp256k1/src/ext.c")
+    base_config.file("depend/secp256k1/contrib/lax_der_parsing.c")
+               .file("depend/secp256k1/src/secp256k1.c")
```

Comment:
We still need ext.c I think unless master has a replacement for `secp256k1_ecdh_raw`